Updated notebooks

omshinde opened the pull request:
Final notebooks with some changes remaining. Good to be shared with the UWG for feedback.

omshinde committed:
updated notebooks

omshinde committed:
executed lonbaord notebook

omshinde committed:
update biomass reference map notebook

wildintellect commented:
@omshinde something is wrong with the Biomass notebook. I think we've had issues like this before, check with the Platform team, or maybe @chuckwondo remembers how to fix when this happens.

omshinde commented:
You're correct. I am pushing the biomass notebook again. There seems to be some issue in it.

omshinde commented:
Here's the notebook demo links - 

1. [Visualization of Cloud-Optimized Earth Observation Data using Lonboard](https://notebooksharing.space/view/104ac04b1111b68694dbc3c3309383cf406186d4f2617f5733dbb7e39a024764#displayOptions=)
2. [Biomass Reference Map Production](https://notebooksharing.space/view/c23674b73f7ff7c605d7f93ce351bcd3fc79e0acec293b111707a8df7a5bde92#displayOptions=)

cc: @wildintellect

omshinde committed:
Commit biomass notebook

omshinde commented:
@wildintellect can you see the biomass notebook now in the nbreview?

omshinde committed:
Add scale bar

omshinde commented:
@wildintellect I've added a hack for the colorbar. Can you please review if this works?

wildintellect commented on lonboard-viz.ipynb:
```diff
@@ -5,7 +5,23 @@
    "id": "56a301bb-74ef-4275-9c3d-419832f38297",
```
<p>I'm confused by the labels on the color ramp. Can we fix those. Also can you put the notebook sharing links in the Readme at the top of the repo for easy access.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/3/discussion/#comment-1597196883'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

omshinde committed:
Adding scale bar

omshinde committed:
Final Lonbaord notebook addressing comments

omshinde committed:
Update biomass notebook

omshinde committed:
Update readme with live links

omshinde commented:
@wildintellect I pushed the notebooks with following updates-

1. Addressed review comments
2. Updated variable names to be intuitive
3. Removed cells with error
4. Updated README

I've not pushed META CHM validation layer to the repo considering David is going to push it but created a rendered link for it.

omshinde committed:
Updating AGB scale values
